src/pages/search.tsx

```diff
@@ -1,55 +1,48 @@
-import Head from "next/head";
-import Link from "next/link";
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { api } from '@/utils/api';
+import { Link } from 'lucide-react';
```

Comment:
oops you imported wrong link component. should be from next/link

Reply:
woops indeed! easy fix